cli/cli/kurtosis_gateway/connection/connection.go

```diff
@@ -147,7 +181,8 @@ func newLocalPortToPodPortConnection(kubernetesRestConfig *k8s_rest.Config, podP
 }
 
 func (connection *gatewayConnectionToKurtosisImpl) Stop() {
-	connection.portforwarder.Close()
+	logrus.Infof("Closing connection to pod: %s", connection.urlString)
+	close(connection.portforwarderStopChannel)
```

Comment:
We close the port forwarder using the stop channel which is the recommended way.